### DISCUSSION TITLE
Docker image pulled incompatible mongodb given skipped major version 7.0

### DISCUSSION THREAD

**Discussion by Originalimoc** [danny-avila/LibreChat#4157] [Troubleshooting]
### What happened?

If db is 6.0 and mongodb is 8.0, startup would fail.
To manually fix this, run these in your docker-compose.yml(aka project root) folder
```
docker run -it --rm --entrypoint /bin/bash -v $(pwd)/data-node:/data/db mongo
apt update
apt install wget
wget https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-ubuntu2204-7.0.14.tgz -O /tmp/mongodb-linux-x86_64-ubuntu2204-7.0.14.tgz
tar -xzf /tmp/mongodb-linux-x86_64-ubuntu2204-7.0.14.tgz -C /tmp
/tmp/mongodb-linux-x86_64-ubuntu2204-7.0.14/mongod --dbpath /data/db --fork --logpath /var/log/mongod.log
mongosh
use admin
db.adminCommand( { setFeatureCompatibilityVersion: "7.0", confirm: true } )
db.adminCommand( { shutdown: 1 } )
.exit
mongod --dbpath /data/db --fork --logpath /var/log/mongod.log
mongosh
use admin
db.adminCommand( { setFeatureCompatibilityVersion: "8.0", confirm: true } )
db.adminCommand( { shutdown: 1 } )
.exit
```
Then restart docker image.
for dev, revert to 7.0 and execute this command then update to 8.0.

### Steps to Reproduce

Start with an early version and docker pull latest image.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Not a bug on our end, we could specify a specific mongodb version but using latest is best.

**Comment by Originalimoc** [comment]
It's mainly confusing for deployment update. I regularly do a docker pull to update since this project switched to using an image by default. So at least starting here/this version could call an update of FCV to 8.0 from webapp run.
